README.md

```diff
@@ -10,6 +10,42 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### File structure ###
+
+You should not have multiple classes in the same file, each class should have its own file. 
+
+The following list should be the standard struct/class organization of all your Swift files, in this specific order:
+
+* Static variables
```

Comment:
What about order of `private / public` static variables? Which comes first? 

My suggestion:

`Public static`
`Private static`